examples/electron/README.md

```diff
@@ -0,0 +1,17 @@
+# React+Monaco+Electron example
+
+## Run
```

Comment:
Maybe add a simple option: "npm run start" as well.

Reply:
fixed in https://github.com/superRaytin/react-monaco-editor/pull/50/commits/6ce01b5cb8f4151deb8689c43e2e429c386fbe0d